File: app/src/main/java/vergecurrency/vergewallet/viewmodel/PinVerificationViewModel.java
package vergecurrency.vergewallet.viewmodel;

import androidx.lifecycle.ViewModel;

import vergecurrency.vergewallet.service.model.PreferencesManager;

public class PinVerificationViewModel extends ViewModel {

	public PinVerificationViewModel(){

	}


	public void setPin(String pin) {
		PreferencesManager.setPin(pin);
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/viewmodel/PinVerificationViewModel.kt
package vergecurrency.vergewallet.viewmodel

import androidx.lifecycle.ViewModel
import vergecurrency.vergewallet.service.model.EncryptedPreferencesManager

class PinVerificationViewModel : ViewModel() {


    fun setPin(pin: String) {
        EncryptedPreferencesManager.pin = pin
    }
}
